### ISSUE TITLE
Cannot read properties of undefined (reading 'presets')

### ISSUE THREAD

**Issue by alanneves** [swagger-api/swagger-ui#7918]
### Q&A (please complete the following information)
 - OS: Windows 11
 - Browser: chrome
 - Version: 99
 - Method of installation: npm
 - Swagger-UI version:`swagger-ui-dist": "3.49.0" `
 - Swagger/OpenAPI version: Swagger 2.0, OpenAPI 3.0

### Content & configuration

Swagger-UI configuration options:
```js
SwaggerUIBundle({
  urls,
  dom_id: '#swagger-ui',
  presets: [SwaggerUIBundle.presets.apis, SwaggerUIStandalonePreset],
  layout: 'StandaloneLayout',
});
```

### Describe the bug you're encountering
I have a simple Vue 3 project that uses Vite to build it. Locally, everything works well.
But when I try to build the application using `vite preview` I get the following error:
`TypeError: Cannot read properties of undefined (reading 'presets')`
Basically, when I'm building the project to put in production I get an error.

### To reproduce...

1. Create a Vue 3 project.
2. Add swagger-ui-dist to the repository
3. Build the project using Vite.

### Expected behavior
I expect to see the documentation

### Screenshots
Error:
![image](https://user-images.githubusercontent.com/38226523/159173614-30d64286-d6a5-4e52-8545-97319b86caac.png)
Success when I run locally:
![image](https://user-images.githubusercontent.com/38226523/159173550-0c705b48-f15c-4679-996c-d579bfe7f32e.png)

**Comment by char0n** [comment]
Hi,

`swagger-ui-dist": "3.49.0"` is not longer supported. Please use `swagger-ui-dist@4.x.y` and reopen this issue if the problem still occurs.

**Comment by ameinhardt** [comment]
@char0n : I have the same issue with vite and 4.10.3. If I remove the try/catch in [swagger-ui-dist/index](https://github.com/swagger-api/swagger-ui/blob/7260f967ec53c8be13ffe0dbe3107bbc6a3766aa/swagger-ui-dist-package/index.js#L1-L11) the code is included in the bundle. Maybe vite/rollup sees that as optional dependency?
If I try to import directly from swagger-ui-dist/swagger-ui-bundle, I get typescript errors

**Comment by mmutavdzic** [comment]
Why this is issue closed?! I face same behaviur on my side with vite and vue(3) - swagger-ui:  4.14.3 - It is working on dev, but not on production

fyi without @ameinhardt  "ignoreTryCatch" settings the error was not thrown. It was just rendering empty #app root element on the swagger page
![image](https://user-images.githubusercontent.com/115781977/195798808-753e6ec3-9bfd-41b2-9037-9fc347d83afb.png)

**Comment by mmutavdzic** [comment]
here is my package.json 
{ <br/>
"name": "testvue", <br/>
 "version": "0.0.0",<br/>
  "scripts": {<br/>
  "dev": "vite  --port 8080",<br/>
  "build": "run-p type-check build-only",<br/>
  "preview": "vite preview --port 8080",<br/>
  },<br/>
  "dependencies": <br/>
    {<br/>
    "swagger-ui": "^4.14.3",<br/>
    "vue": "^3.2.38"<br/>
  },<br/>
  "devDependencies":<br/> 
{<br/>
    "@types/node": "^16.11.56",<br/>
    "@types/swagger-ui": "^3.52.0",<br/>
    "@vitejs/plugin-vue": "^3.0.3",<br/>
    "@vitejs/plugin-vue-jsx": "^2.0.1",<br/>
    "@vue/tsconfig": "^0.1.3",<br/>
    "npm-run-all": "^4.1.5",<br/>
    "typescript": "~4.7.4",<br/>
    "vite": "^3.0.9",<br/>
    "vue-tsc": "^0.40.7"<br/>
  }<br/>
}`<br/>


and my App.vue

<script lang="ts"><br/>
import { defineComponent } from "vue";<br/>
import SwaggerUI from "swagger-ui";<br/>
import "swagger-ui/dist/swagger-ui.css";<br/>
import spec from "../swager.json";<br/>
export default defineComponent({<br/>
  mounted() {<br/>
    SwaggerUI({<br/>
      spec: spec,<br/>
      dom_id: "#swagger",<br/>
    });<br/>
  },<br/>
});<br/>
</script><br/>
**template**<br/>
  **main**<br/>
    **div class="swagger" id="swagger"    div**<br/>
  **main**<br/>
**template**<br/>

**Comment by linspw** [comment]
Same problem

**Comment by xieyuwu** [comment]
Same problem, here is my code:
<template>
    <div class="importSwagger" style="width: 100%;margin: 10px auto;text-align: text">
        <div id="swagger-ui">

        </div>
    </div>
</template>

<script>
    export default {
        name: 'ApiDocInfo'
    }
</script>

<script setup>
    import { SwaggerUIbundle, SwaggerUIStandalonePreset } from 'swagger-ui-dist';
    import { onMounted } from 'vue';
    console.log(SwaggerUIbundle);

    onMounted(() => {
        const ui = SwaggerUIbundle({
            url: "https://petstore.swagger.io/v2/swagger.json",
            dom_id: '#swagger-ui',
            presets: [
                SwaggerUIbundle.presets.apis,
                SwaggerUIStandalonePreset,
            ],
            layout: 'StandaloneLayout',
        })
    })
</script>

![image](https://user-images.githubusercontent.com/30075943/202338867-a9360d35-bf24-4eed-857c-3ec770eed0e6.png)

**Comment by nandar-dev** [comment]
> 

It's work for me. Thanks.

**Comment by JulianCataldo** [comment]
Got this too:

`TypeError: Cannot read properties of undefined (reading 'isRequired')`

With
`"swagger-ui": "^4.15.5"` and `"astro": "^1.6.12"` (`vite: 3.2.5`)

Works great in dev, but empty `div` in production.

**Comment by pawel-omniaz** [comment]
I've got exactly same error:

`TypeError: Cannot read properties of undefined (reading 'isRequired')`

Only after build for production :/
